shipyard/README.md

```diff
@@ -0,0 +1,60 @@
+# Shipyard
+
+## Overview
+
+Shipyard unblocks DevOps teams from bulky, outdated production pipelines. We resolve the pre-production or staging bottleneck with our agile on-demand ephemeral (testing or preview) environments.
+
+The Datadog Shipyard integration automatically passes your [Shipyard][1] ephemeral environment logs to your Datadog dashboard. For every application you have enables the Datadog logs for, Shipyard sends the runtime logs of your application to your dashboard. 
+
+This is useful for: 
+
+- monitoring your application’s runtime status
+- using Datadog to enrich and parse your runtime logs
+- uncover trends and compare the logs from your application’s environments using Datadog’s easy to use interface
+
+## Setup
+
+These instructions assume you have a Shipyard account and application set up. If you do not, you can get started [for free][2]. For more information about setting up the Datadog integration, see the [Shipyard documentation][7]. 
```

Comment:
Should this sentence:
> For more information about setting up the Datadog integration, see the [Shipyard documentation][7]. 

go at the bottom of the section?

Reply:
```suggestion
These instructions assume you have a Shipyard account and application set up. If you do not, you can get started [for free][2]. 
```